src/components/Switch.js

```diff
@@ -1,16 +1,21 @@
 import React, {Component} from 'react';
-import {TouchableOpacity, Animated} from 'react-native';
+import {Animated} from 'react-native';
 import PropTypes from 'prop-types';
 import styles from '../styles/styles';
+import * as Pressables from './Pressable';
 
 const propTypes = {
     /** Whether the switch is toggled to the on position */
     isOn: PropTypes.bool.isRequired,
 
     /** Callback to fire when the switch is toggled */
     onToggle: PropTypes.func.isRequired,
+
+    /** Accessibility label for the switch */
+    accessibilityLabel: PropTypes.string.isRequired,
```

Comment:
`accessibilityLabel` is declared as Required but it's value is undefined as it not passed from components using `Switch`.

Reply:
Filled, good catch
Thanks

Reply:
@robertKozik `accessibilityLabel` should be added in `TimezoneInitialPage` as well.

https://github.com/Expensify/App/blob/e52c1e295713a9f67aeb69ba82d011636d8830d6/src/pages/settings/Profile/TimezoneInitialPage.js#L61-L64